packages/snap/src/Snap.service.ts

```diff
@@ -339,69 +314,74 @@ class SnapService {
   ): Promise<VerifiableCredential[]> {
     const { credentialOffer } = args;
 
-    if (credentialOffer.startsWith('openid-credential-offer://')) {
-      await VeramoService.importIdentifier();
-      return [
-        await VeramoService.handleOIDCCredentialOffer({
-          credentialOfferURI: credentialOffer,
-        }),
-      ];
-    }
+    if (await UIService.snapConfirm(handleCredentialOfferContent())) {
```

Comment:
Maybe better to first check if the credential offer is valid and then show a popup.
Also good to show some information in the popup

Reply:
Same thing for OIDC credential offers